addons/mail/static/src/models/attachment/attachment.js

```diff
@@ -328,6 +345,18 @@ function factory(dependencies) {
             inverse: 'attachments',
         }),
         type: attr(),
+        /**
+         * AbortController instance used to upload that attachment to the server.
+         * It's used to have an access to the AbortSignal and allow a control on
+         * the attachment uploading process.
+         */
+        uploadController: attr({
```

Comment:
Would rename field `uploadingAbortController`. It's a more appropriate name, based on what this does conceptually (a controller to abort uploading).